.editorconfig

```diff
@@ -0,0 +1,21 @@
+# editorconfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+indent_style = space
+indent_size = 2
+
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Is there any reason we can't trim whitespaces on `.md` files? If not, I'd rather keep trimming on.

Reply:
yes totally agree in fact